.devcontainer/devcontainer.json

```
@@ -0,0 +1,46 @@
+// For more details, see https://aka.ms/devcontainer.json.
+{
+  "name": "Sofort Pay",
+  "workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}/",
+  "image": "mcr.microsoft.com/devcontainers/javascript-node:20",
+  "postCreateCommand": "npm install --ignore-scripts",
+  "postAttachCommand": {
+    "Start server": "npx kill-port 8080 && npm start"
```

flag can be added to install `kill-port` directly

```suggestion
    "Start server": "npx kill-port --yes 8080 && npm start"
```